kip-0037.md

```diff
@@ -12,123 +12,114 @@ Created: 2025-02-17
 
 ## Abstract
 
-We propose the `kadena_getAccount_v1` method to provide an active account known by the wallet. This allows dApps to present the user with the selected account and allow them to use it for creating and signing transactions.
+This KIP standardizes the `kadena_getAccount_v1` method to provide the active
+account known by the wallet. This allows dApps to present the user with the
+currently selected account and use it for creating and signing transactions.
 
 ## Motivation
 
-The `kadena_getAccount_v1` method provides a standardized way to retrieve a list of accounts from a Kadena wallet.
+The `kadena_getAccount_v1` method provides a standardized way to retrieve a
+single, active account from a Kadena wallet. This method is used when the dApp
+does not need to specify which account to retrieve and relies on the wallet's
+current active selection.
 
 ## Specification
 
 ### Method Definition
 
 **Method Name:** `kadena_getAccount_v1`
 
-### Request Parameters
+#### Parameters
 
-```typescript
-interface GetAccountRequest {
-  account?: {
-    publicKey: string
-    contract: string
```

Comment:
Comment from @javadkh2: when the dapp knows which contract to use, they should be able to select it. It's about the right UX for the user so they don't have to think about it. It can be optional though